.circleci/config.yml

```diff
@@ -923,6 +923,7 @@ jobs:
           name: "Create a file contains test names"
           working_directory: ~/workbench/e2e
           command: yarn jest --listTests | grep -o 'tests/.*.ts$' > e2e_tests.txt
+      - gcloud-auth-login-stored-credentials: # needed to run project.rb generate-impersonated-user-tokens as part of "yarn test:ci"
```

Comment:
I see that the [PR that you referenced also had `with_application_default_credentials: true`](https://github.com/all-of-us/workbench/blob/78a82d3eddae5da31139bbf3aa14b97986cf8be0/.circleci/config.yml#L327)

Does this need to be here too?

Reply:
I would expect no, because that is specifically to upload the test results.  Uploading the test results now has an explicit auth with the default credentials enabled